Makefile

```diff
@@ -1035,8 +1034,6 @@ savedefconfig: $(KCONFIG_DIR)/conf
 	@$(COMMON_CONFIG_ENV) $< \
 		--savedefconfig=$(if $(DEFCONFIG),$(DEFCONFIG),$(CONFIG_DIR)/defconfig) \
 		$(CONFIG_CONFIG_IN)
-	@$(SED) '/UK_DEFCONFIG=/d' $(if $(DEFCONFIG),$(DEFCONFIG),$(CONFIG_DIR)/defconfig)
```

Comment:
This line is removed because it produces an unnecessary console output, it does not modify the resulting file. Additionally, `UK_DECONFIG=` was never stored to the `defconfig` file in my tests.